### DISCUSSION TITLE
I require assistance in assessing the compatibility of applications with Debian 12. Please

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#2034] [Announcements]
Complete! Thanks to everyone who helped!!

**Comment by AlexGeek77** [comment]
No issues installing jackett on Debian 12.  https://pastebin.com/9imXjSGp
No issues installing transmission on Debian 12. https://pastebin.com/4xY2ZzLC 
No issues installing autobrr on Debian 12.  https://pastebin.com/uajNkAad

**Reply by tteck** [reply]
Thank you.

**Comment by AlexGeek77** [comment]
No issue installing homer on Debian 12. https://pastebin.com/t7d0q2d0 
**_Errors_** installing Blocky on Debian 12. https://pastebin.com/KGDVmGuY 
No issue installing Cronicle on Debian 12. It appears to be running but I've never used the program so I can't tell if everything is 100% working. I can get to the Cronicle login screen and `root` is running Chronicle Server, so that seems good.  https://pastebin.com/hwkyuwUp

**Reply by tteck** [reply]
Your help is appreciated.

**Reply by tteck** [reply]
fixed Blocky

**Reply by AlexGeek77** [reply]
Blocky works now on Debian 12 for me.
```
root@blocky:~# systemctl status blocky
* blocky.service - Blocky
     Loaded: loaded (/etc/systemd/system/blocky.service; enabled; preset: enabled)
     Active: active (running) since Mon 2023-11-13 14:03:07 EST; 1min 37s ago
   Main PID: 1047 (blocky)
      Tasks: 7 (limit: 76691)
     Memory: 15.9M
        CPU: 2.068s
     CGroup: /system.slice/blocky.service
             `-1047 /opt/blocky/./blocky --config config.yml

Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:   numCPU =       1
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:   numGoroutine = 15
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:   memory:
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:     alloc =                 7 MB
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:     heapAlloc =             7 MB
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:     sys =                  55 MB
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server:     numGC =               310
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server: Starting server
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server: UDP server is up and running on address :553
Nov 13 14:03:09 blocky blocky[1047]: [2023-11-13 14:03:09]  INFO server: TCP server is up and running on address :553
```

**Comment by onethree7** [comment]
nexctcloudpi does not work.
https://pastebin.com/kcESpEm2

**Reply by tteck** [reply]
Thanks

**Reply by onethree7** [reply]
whoogle works
https://pastebin.com/qyZcUKn0

**Reply by tteck** [reply]
Thank you.

**Reply by onethree7** [reply]
most welcome! thank YOU!

CasaOS works!
https://pastebin.com/Lbfcwnqb

**Reply by onethree7** [reply]
daemonsync works too
https://pastebin.com/y9hAaiSP

**Reply by onethree7** [reply]
prometheus works too!
https://pastebin.com/2yHbbCy9

**Comment by t0rb3n** [comment]
If green webinterface is reachable and appears to work. 
If red installation failed.

✅ go2rtc - https://pastebin.com/HbFARkxu
✅ homebridge - https://gist.github.com/t0rb3n/5a07c38d8941cdaff918333d45143339
❌ hyperion - https://pastebin.com/kXtH2hgx (dependencies `libpython3.10` and `libmbedtls12` not installable)
✅ iobroker - https://gist.github.com/t0rb3n/fada84ea49ccdf76d834d1e38a1b2960
✅ motioneye - https://pastebin.com/kkKNDEhD
✅ navidrome - https://pastebin.com/P2ryrQ7C
✅ octoprint - https://pastebin.com/7xFXa5rt 
✅ rtsptoweb - https://pastebin.com/5tk4gkv3
✅ syncthing - https://pastebin.com/QReez6CV
✅ Whisparr - https://pastebin.com/MpGGvKGS
❌ wireguard - https://pastebin.com/ny2sEM9Q (`wireguard-dkms` no installation candidate)
❌ Yunohost - https://pastebin.com/re2EK5m7 (yunohost not available for debian 12)

**Reply by tteck** [reply]
Thank you very much!
